.eslintrc.js

```diff
@@ -38,7 +38,6 @@ module.exports = {
     // Remove this when m-c updates their eslint: See https://github.com/mozilla/activity-stream/pull/4219
     "RPMSendAsyncMessage": true,
     "NewTabPagePreloading": true,
-    "DOMLocalization": true,
```

Comment:
Why did this disappear?

Reply:
It was added to eslint-plugin-mozilla as part of 1.3.0, so it's redundant here